.github/workflows/main.yml

```diff
@@ -258,3 +258,37 @@ jobs:
           tag_name: ${{ steps.stax.outputs.tag_name }}
           draft: false
           prerelease: false
+
+  build_package_flex:
+    needs: [configure, build, build_ledger, test_zemu]
+    if: ${{ github.ref == 'refs/heads/main' }}
+    runs-on: ubuntu-latest
+    container:
+      image: zondax/ledger-app-builder:latest
+      options: --user ${{ needs.configure.outputs.uid_gid }}
+      env:
+        BOLOS_SDK: /opt/flex-secure-sdk
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Install deps
+        run: pip install ledgerblue
+
+      - name: Build Flex
+        shell: bash -l {0}
+        run: make SUBSTRATE_PARSER_FULL=1
```

Comment:
Sorry for one more comment, but just notice, we do not have the PRODUCTION_BUILD = 0 whe we are building the app for the packages. Please add it in all the build_package_<device>.
 This way, we make sure the installers we are releasing have the DEMO tag on it and are not official release.